webextension/scripts/background.js

```diff
@@ -312,6 +312,7 @@ function getCachedTvNews(url, onSuccess, onFail) {
 }
 
 function getCachedFactCheck(url, onSuccess, onFail) {
+  // require "https://data.our.news/api/*" in manifest permissions
```

Comment:
We won't be using this service going forward, even when we reinstate the fact-checking feature. So this comment is no longer relevant.